MigrationTutorial/Models/V3/MachinaryAndTool.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using MongoDB.Bson;
+using Realms;
+
+namespace MigrationTutorial.Models.V3
+{
+    public class MachinaryAndTool : RealmObject
```

Comment:
```suggestion
    public class MachineryAndTool : RealmObject
```

I can see `ToolName` later on but where is the Machinery part? Would it make sense to split them into two classes, maybe having a common base class?

Reply:
In my mind a tool and a machine serve the same purpose (help you do some work), suffer from issues hence maintenance is needed, are addressable with a name and have a brand. Because of all this I did not see the need for 2 different classes with exactly the same fields.